### DISCUSSION TITLE
State of theme-ui question

### DISCUSSION THREAD

**Discussion by maiertech** [system-ui/theme-ui#1354] [Q&A] [answered]
I have been using `theme-ui` for quite some time and think it's a great choice for anyone looking for a CSS-in-JS (plus so much more) solution. I remember when @jxnblk's side project graduated and received quite a bit of attention and staff resources from @gatsbyjs and I assumed they would position Theme UI to be the official way of doing styling for Gatsby sites and themes. At one point I saw even @mxstbr being a staff contributor to Theme UI.

But then things seemed to calm down a bit and the project seems to have transitioned into a community project. That's not a bad thing since Theme UI is very active and keeps evolving as a solid community project. But I would be curious to hear more what the reasons were for this shift and how Theme UI positions itself relative to the Gatsby ecosystem.

**Comment by lachlanjc** [comment, marked as answer]
Yeah, so as you said Brent wrote the original. He's left the social internet (GitHub/Twitter/Instagram) as of this summer for at least a hiatus, & is no longer working on the project day-to-day. I haven't spoken to anyone at Gatsby about their plans. So for now, we fully support Gatsby & integrate with it but as far as the company goes I'm not aware of any formal intentions.

Brent handed publishing & merging over to @hasparus, who has been doing great work keeping maintenance going for a few months. In the last few weeks, I've joined the maintenance team (as a side project, this is unpaid for all of us) working especially on docs, & releases have sped up. I'm not sure how @atanasster wants to be labeled, but he's been putting in great work as well on a number of bug fixes & new features recently, plus building some killer experimental new docs. The 3 of us have become the primary maintainers of the library (again, as a side gig), plus a few other frequent contributors in the Discord. While none of us have a ton of time to commit to the project, we want to keep improving the features & performance, docs & marketing, upgrades & maintenance on the library. We welcome all help & volunteers, since we're just superfans talking & trying to figure out the best path forward, & while we have a general roadmap we don't have specific deadlines or time commitments.

Does that help? Happy to chat more!

**Reply by atanasster** [reply]
ok - my "profile" :)
I am a so-called 'heavy' user of theme-ui on multiple projects with gatsby and nextjs. Also maintaining a library of additional components for theme-ui. Basically I have invested a lot of my time into theme-ui.
I started contributing more recently as I was getting worried that theme-ui will become an(other) abandoned project, but now I see the energy of other contributors such as @hasparus @lachlanjc @dcastil as well as some of the older PRs that we are getting through right now, and that keeps me motivated to also give theme-ui some of my time.

My philosophy is that every open-source contribution - giving feedback, testing new releases, filing an issue, opening a PR is a gift from the contributor and should be appreciated as such. Also I dont believe much in the "benevolent dictator" way of conducting projects and hope more people come forward to shape theme-ui.
